layers/+lang/ess/README.org

```diff
@@ -3,57 +3,63 @@
 [[file:img/r.jpg]]
 
 * Table of Contents                                         :TOC_4_org:noexport:
- - [[#install][Install]]
- - [[#options][Options]]
-   - [[#style][Style]]
-   - [[#roxygen][Roxygen]]
- - [[#key-bindings][Key Bindings]]
-   - [[#inferior-ess-processes][Inferior ESS Processes]]
-   - [[#evaluation-commands][Evaluation Commands]]
-   - [[#Developer][Developer]]
-   - [[#debugging][Debugging]]
-   - [[#help][Help]]
+ - [[Install][Install]]
+ - [[Options][Options]]
+   - [[Style][Style]]
+   - [[Roxygen][Roxygen]]
+ - [[Key Bindings][Key Bindings]]
+   - [[Process Interaction][Process Interaction]]
+   - [[Evaluation Commands][Evaluation Commands]]
+   - [[Developer][Developer]]
+   - [[Documentation][Documentation]]
+   - [[Debugging][Debugging]]
+   - [[Views][Views]]
+   - [[ESS Key Bindings][ESS Key Bindings]]
+     - [[Inferior ESS Process Keybindings][Inferior ESS Process Keybindings]]
+     - [[ESS Watch][ESS Watch]]
+     - [[ESS Rdired][ESS Rdired]]
+     - [[ESS Data View][ESS Data View]]
 
 * Install
-To use this configuration layer, add it to your =~/.spacemacs=. You will need to
-add =ess= to the existing =dotspacemacs-configuration-layers= list in this
-file.
+  To use this configuration layer, add it to your =~/.spacemacs=. You will need to
```

Comment:
Spurious change. Only changes whitespace (indentation). Same for many of the hunks below this one.